webapi/src/main/scala/org/knora/webapi/responders/v2/ResourceUtilV2.scala

```diff
@@ -133,27 +140,47 @@ object ResourceUtilV2 {
     } yield defaultObjectAccessPermissionsResponse.permissionLiteral
 
   /**
-   * Checks whether a list node exists, and throws [[NotFoundException]] otherwise.
+   * Checks whether a list node exists and if is a root node.
```

Comment:
could you maybe also add a description of the return type?

Reply:
Added `@return` type.